lib/livereload.js

```diff
@@ -97,6 +97,12 @@ function StartServer() {
                         return monkeyPatch() + snippet + match;
                     }
                 }
+            },
+            minify: false,
```

Comment:
These are default values - why does it make a difference? http://www.browsersync.io/docs/options/